stl/inc/xtree

```diff
@@ -328,13 +328,14 @@ struct _Tree_node {
     _Nodeptr _Right; // right subtree, or largest element if head
     char _Color; // _Red or _Black, _Black if head
     char _Isnil; // true only if head (also nil) node; TRANSITION, should be bool
-    value_type _Myval; // the stored value, unused if head

```

Comment:
Similarly, I believe that this comment has value (at least the "unused if head" part).